.prow.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -xeuo pipefail
+
+yum -y install jq
+for jsonfile in *.json; do
+    echo "Checking JSON syntax for $jsonfile"
+    jq < $jsonfile . >/dev/null
```

Comment:
The `.` here is needed to work around a bug where `jq` will refuse to work if redirecting output to something that's not a TTY: https://github.com/stedolan/jq/issues/1351.